src/_guides/whats-new.md

```diff
@@ -17,7 +17,51 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://medium.com/dartlang
 
+## February 3, 2022: 2.16 release
+
+This section lists notable changes made from December 8, 2021,
+through February 3, 2022.
+For details about the 2.16 release, 
+see [Dart 2.16: Improved tooling and platform handling][].
+
+[Dart 2.16: Improved tooling and platform handling]: https://medium.com/dartlang/dart-2-16-improved-tooling-and-platform-handling-dd87abd6bad1
+
+### Docs updated or added to dart.dev
+{: .no_toc}
+
+We [updated the website infrastructure][] to a Docker based setup
+to enable easier contributions and more closely align with
```

Comment:
Maybe link `easier contributions` to something like https://github.com/dart-lang/site-www#getting-started ?